MiniProjectUIKit/Component/RecipeCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  RecipeCollectionViewCell.swift
+//  MiniProjectUIKit
+//
+//  Created by Vincent Saranang on 06/12/24.
+//
+
+import UIKit
+
+class RecipeCollectionViewCell: UICollectionViewCell {
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let areaBadge = UILabel()
```

Comment:
selalu declare type-nya apa, bisa ngurangin beban compilation

```suggestion
    private let imageView: UIImageView = UIImageView()
    private let titleLabel: UILabel = UILabel()
    private let areaBadge: UILabel = UILabel()
```